src/libraries/System.Text.Json/tests/Serialization/StreamTests.cs

```diff
@@ -0,0 +1,355 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+using Xunit;
+
+namespace System.Text.Json.Serialization.Tests
+{
+    public static partial class StreamTests
+    {
+        // To hit all possible continuation positions inside the tested object,
+        // the outer-class spacer needs to be between 5 and 116 bytes long.
+
+        // {"S":"xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx","C":{"A":null,"B":"Hello","C":42,"D":null,"E":3.14E+17,"F":null,"G":true,"H":null,"I":[42,17],"J":{"A":null,"B":7}}}
+        // |<------------------------------------------------128 byte buffer------------------------------------------------------------->|
+        // {"S":"xxxxx","C":{"A":"Hello","B":null,"C":42,"D":null,"E":3.14E+17,"F":null,"G":true,"H":null,"I":[42,17],"J":{"A":null,"B":7}}}
+        public static IEnumerable<object[]> ContinuationSpacerLengths
+            => Enumerable.Range(5, 116 - 5 + 1).Select(length => new object[] { length });
```

Comment:
How much longer does it take to run all the project tests in both `Debug`/`Release` modes as a result of using this value? If it's a lot, can we reduce the number of lengths returned here without losing the test coverage?

Reply:
After doubling all new tests because of `IgnoreNullValues=true|false` these are the local run results for target `net5.0`
||base|with continuation|
|:-:|:-:|:-:|
|**Debug**|`Total: 8706, Errors: 0, Failed: 0, Skipped: 0, Time: 70,223s`|`Total: 9826, Errors: 0, Failed: 0, Skipped: 0, Time: 72,229s`|
|**Release**|`Total: 8706, Errors: 0, Failed: 0, Skipped: 0, Time: 28,578s`|`Total: 9826, Errors: 0, Failed: 0, Skipped: 0, Time: 31,961s`|

I would say 2-3 seconds is good enough. How would you decrease the tested positions without knowing what could go wrong in advance?
BTW I cannot run the tests for target `net461` from 6.0 branch locally because I get tons of errors. I "hope" it's just a glitch with my machine
```
System.IO.FileLoadException : Die Datei oder Assembly "System.Runtime.CompilerServices.Unsafe, Version=4.0.4.1, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" oder eine Abhängigkeit davon wurde nicht gefunden. Fehler bei der Überprüfung des starken Namens. (Ausnahme von HRESULT: 0x8013141A)
```

Reply:
I think 2-3s is worth the coverage here.

---

> cannot run the tests for target net461

Did you try building the libraries from the repo root before running the tests?

`build -s clr+libs -rc Release -f net461`

Reply:
`-s` is for signing? I'll give it a try. Can't build clr at all after upgrade to VS 16.8 preview 3

Reply:
Okay `-s` is `-subset` but optional. I rebuilt everything and still cannot run tests for net461. I would say it fails since #41616 got merged. The binding redirect is generated in S.T.J.Tests.dll.config but has no effect during the test run in VS nor with `dotnet build /t:test`.

Reply:
Opened #42250 but still don't know if it's only my environment.

Reply:
Thanks for filing the issue. It looks like there was indeed an issue which is being looked into.